aiven/client/cli.py

```diff
@@ -1339,7 +1339,9 @@ def service__user_password_reset(self):
             password=self.args.new_password,
         )
 
-    @arg.project
+    @arg(
+        "--project", help="Project name to use, default %(default)r", default=os.environ.get("AIVEN_PROJECT"), required=True
+    )
```

Comment:
Instead of changing the argument it looks like the correct fix would be to change `self.args.project` to `self.get_project()` few lines down. Same for the other change.

Reply:
Ah ha, thanks! Branch updated